README.md

```
@@ -22,7 +22,7 @@ SDK docs can be found [here](./sdk/README.md)
 
 # Example Bot Implementations
 
-Example bots (makers, liquidators, fillers, etc) can be found [here](https://github.com/drift-labs/keeper-bots-v2)
+Example bots (makers, liquidators, fillers, etc) can be found [here](https://github.com/velocity-exchange/keeper-bots-v2)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)